study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/translation/TranslationsProcessingType.java

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.ddp.studybuilder.translation;
+
+/**
+ * Defines different types of translations processing (translation references generation).
+ */
+public enum TranslationsProcessingType {
+
+    /**
+     * Not to run the translations processing.
+     * This is how the conf files processed an old way: translations
+     * should be defined inside variables in templates definitions.
+     */
+    NOT_PROCESS,
```

Comment:
I removed the NOT_PROCESS. It is really not needed.
So only 2 possible values: PROCESS_ALL_TEMPLATES, PROCESS_IGNORE_TEMPLATES_WITH_TRANSLATIONS.
But it looks like that it is enough only one: PROCESS_IGNORE_TEMPLATES_WITH_TRANSLATIONS...
But may be it could be possible to use PROCESS_ALL_TEMPLATES in some future when revisioning will be supported and updates in conf files could be applied to a DB..